### PR TITLE
Fixes issue 7505 (Rename Spagetti to Spaghetti)

### PR DESCRIPTION
**Pull request by alexkar598**
i did not use github editor this time...
fixes: #7505 
🆑 alexkar598
fix: Rename Spagetti to Spaghetti
/🆑

### CODE REVIEW

File: code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -1013,10 +1013,10 @@
 	filling_color = "#C4BF76"
 	list_reagents = list("nutriment" = 2)
 
-/obj/item/weapon/reagent_containers/food/snacks/spagetti
-	name = "Spagetti"
+/obj/item/weapon/reagent_containers/food/snacks/spaghetti
+	name = "Spaghetti"
 	desc = "A bundle of raw spaghetti."
-	icon_state = "spagetti"
+	icon_state = "spaghetti"
```
Comment (Fox-McCloud) [inline code/modules/food_and_drinks/food/snacks.dm]: if you're going to change this, change the icon name.

Comment (taukausanake) [inline code/modules/food_and_drinks/food/snacks.dm]: I have to know who spelled it like this now. Sure I can't spell worth a damn either without spellcheck half the time but really. Who did this?

**Comment by scrubmcnoob** [comment]
knees weak, arms sweaty

**Comment by Fox-McCloud** [comment]
Going to give you 2 days to fix this properly.

**Comment by alexkar598** [comment]
finally got git to work should learn it one day....

**Comment by alexkar598** [comment]
from what i can tell the issue is from before praradise station was created

**Pull request merged**